[2021-11-27 06:15:09,851] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-26T23:14:00+00:00 [queued]>
[2021-11-27 06:15:09,856] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-26T23:14:00+00:00 [queued]>
[2021-11-27 06:15:09,856] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 06:15:09,856] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-11-27 06:15:09,856] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 06:15:09,862] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): upload> on 2021-11-26 23:14:00+00:00
[2021-11-27 06:15:09,865] {standard_task_runner.py:52} INFO - Started process 2087 to run task
[2021-11-27 06:15:09,872] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'elt', 'upload', 'scheduled__2021-11-26T23:14:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/processing.py', '--cfg-path', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmp0_pwnond', '--error-file', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpxfl3l37p']
[2021-11-27 06:15:09,874] {standard_task_runner.py:77} INFO - Job 30: Subtask upload
[2021-11-27 06:15:09,902] {logging_mixin.py:109} INFO - Running <TaskInstance: elt.upload scheduled__2021-11-26T23:14:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-11-27 06:15:09,927] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=elt
AIRFLOW_CTX_TASK_ID=upload
AIRFLOW_CTX_EXECUTION_DATE=2021-11-26T23:14:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-26T23:14:00+00:00
[2021-11-27 06:15:09,928] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T
[2021-11-27 06:15:09,929] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/Users/Home/Airflow/tasks/cleaning.py /Users/Home/Airflow/output worldbank']
[2021-11-27 06:15:09,934] {subprocess.py:85} INFO - Output:
[2021-11-27 06:15:09,987] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_NV.AGR.TOTL.ZS_DS2_en_csv_v2_3160861.csv: No such file or directory
[2021-11-27 06:15:09,994] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.IRIG.AG.ZS_DS2_en_csv_v2_3163914.csv: No such file or directory
[2021-11-27 06:15:09,999] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.ARBL.ZS_DS2_en_csv_v2_3161161.csv: No such file or directory
[2021-11-27 06:15:10,004] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.TRAC.ZS_DS2_en_csv_v2_3163924.csv: No such file or directory
[2021-11-27 06:15:10,009] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.YLD.CREL.KG_DS2_en_csv_v2_3163929.csv: No such file or directory
[2021-11-27 06:15:10,014] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.ARBL.HA.PC_DS2_en_csv_v2_3163892.csv: No such file or directory
[2021-11-27 06:15:10,019] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.AGRI.ZS_DS2_en_csv_v2_3163149.csv: No such file or directory
[2021-11-27 06:15:10,020] {subprocess.py:89} INFO - API_NV.AGR.TOTL.ZS_DS2_en_csv_v2_3160861.csv
[2021-11-27 06:15:10,020] {subprocess.py:89} INFO - API_AG.LND.IRIG.AG.ZS_DS2_en_csv_v2_3163914.csv
[2021-11-27 06:15:10,020] {subprocess.py:89} INFO - API_AG.LND.ARBL.ZS_DS2_en_csv_v2_3161161.csv
[2021-11-27 06:15:10,020] {subprocess.py:89} INFO - API_AG.LND.TRAC.ZS_DS2_en_csv_v2_3163924.csv
[2021-11-27 06:15:10,020] {subprocess.py:89} INFO - API_AG.YLD.CREL.KG_DS2_en_csv_v2_3163929.csv
[2021-11-27 06:15:10,021] {subprocess.py:89} INFO - API_AG.LND.ARBL.HA.PC_DS2_en_csv_v2_3163892.csv
[2021-11-27 06:15:10,021] {subprocess.py:89} INFO - API_AG.LND.AGRI.ZS_DS2_en_csv_v2_3163149.csv
[2021-11-27 06:15:10,024] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-27 06:15:10,043] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=elt, task_id=upload, execution_date=20211126T231400, start_date=20211126T231509, end_date=20211126T231510
[2021-11-27 06:15:10,095] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-27 06:15:10,123] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
